pkgs/by-name/za/zarf/package.nix

```
@@ -53,8 +82,11 @@ buildGoModule rec {
   meta = with lib; {
     description = "DevSecOps for Air Gap & Limited-Connection Systems. https://zarf.dev";
     mainProgram = "zarf";
-    homepage = "https://github.com/defenseunicorns/zarf.git";
+    homepage = "https://github.com/zarf-dev/zarf.git";
```

```suggestion
    homepage = "https://zarf.dev/";
```
We prefer the landing page here, when possible.